models/migrations/v1_22/v285_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_22 //nolint
+
+import (
+	"testing"
+
+	"code.gitea.io/gitea/models/migrations/base"
+
+	"github.com/stretchr/testify/assert"
+	"xorm.io/xorm"
+)
+
+func PrepareOldRepository(t *testing.T) (*xorm.Engine, func()) {
```

Comment:
well now the test only need to cover

https://github.com/go-gitea/gitea/pull/23894/files#diff-b53844cbfcd3620ba577df26583953168d1da604d690fde38f1352045a46e7a2R16-R26

e.g. the ExpandHashReferencesToSha256 part

Reply:
@AdamMajer I would leave that for you if it's ok?

Reply:
I think we don't need to worry about tests covering ALTER TABLE ... ALTER COLUMN as those SQL commands fail hard on errors. They also are not altering any data.

UPDATE is worthy to be tested because sometimes you can mistype something and you get no direct errors.